### PR TITLE
[zh] Update volumes.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh] Update volumes.md

### CODE REVIEW

File: content/zh/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -1601,15 +1602,13 @@ For more information about StorageOS, dynamic provisioning, and PersistentVolume
 关于 StorageOS 的进一步信息、动态供应和持久卷申领等等，请参考
 [StorageOS 示例](https://github.com/kubernetes/examples/blob/master/volumes/storageos)。
 
-### vsphereVolume {#vspherevolume}
+### vsphereVolume (deprecated) {#vspherevolume}
```
Comment (Sea-n) [inline content/zh/docs/concepts/storage/volumes.md:1605]: Maybe it should be translated to `### vsphereVolume （已棄用） {#vspherevolume}`.

Comment (Sea-n) [reply content/zh/docs/concepts/storage/volumes.md]: Hey @my-git9, your fix-up commit seems to put the English comment in the wrong place. 😅

BTW, are you in the [Kubernetes Slack](https://slack.k8s.io)? We can inform you about small things like this.

Comment (my-git9) [reply content/zh/docs/concepts/storage/volumes.md]: yes

**Comment by tengqm** [comment]
@my-git9 Please coordinate with @mengjiao-liu since this page is captured in #33476 as part of 'concepts-3'.

**Pull request closed without merging**